src/block/decompress.rs

```diff
@@ -276,15 +278,15 @@ fn decompress_internal<const USE_DICT: bool>(
             debug_assert!(unsafe { start_ptr.add(match_length) } <= output_end);
 
             // In this branch we know that match_length is at most 18 (14 + MINMATCH).
-            // But the blocks can overlap, so make sure they are at least 20 bytes apart
-            // to enable an optimized non-overlaping copy of 20 bytes.
-            if offset < 20 {
```

Comment:
I chose here 20, because it's dividable by 4, but I didn't actually run benchmarks between 18 and 20 for it. Just FYI

Reply:
I think it was 16 w/ a branch to 24 before. I got improvements switching to a fixed 20, but fixed 18 should be equaliy sensible.